lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -218,7 +232,96 @@ private function relationshipExists($sourceActorId, $targetActorId, $relationTyp
       ':type_id' => $relationTypeId
     ];
 
-    return QubitPdo::fetchOne($sql, $params) !== false
-      || QubitPdo::fetchOne($sql, $paramsVariant) !== false;
+    if ($relationId = QubitPdo::fetchColumn($sql, $params))
+    {
+      return $relationId;
+    }
+
+    $relationId = QubitPdo::fetchColumn($sql, $paramsVariant);
+
+    return !empty($relationId) ? $relationId : false;
```

Comment:
I think `return $relationId;` is equivalent to `!empty($relationId) ? $relationId : false;` here, since Qubit::fetchOne() will return false if no results are found.

Reply:
Hi @mcantelon, sorry if I wasn't clear with my first comment but I think you can just use `return $relationId;` for line 242.  QubitPdo::fetchColumn() returns false if their are no rows in the resultset, so there's no need to check `!empty()`, $relationId will either have value, or be false.

Reply:
Oh, nevermind. You fixed this in the next commit. 😊